prereise/gather/winddata/hrrr/hrrr_api.py

```diff
@@ -0,0 +1,115 @@
+import requests
+from pandas import date_range
+
+from prereise.gather.winddata.hrrr.grib import GribRecordInfo
+from prereise.gather.winddata.hrrr.helpers import (
+    formatted_filename,
+    generate_index_url,
+    get_indices_that_contain_selector,
+)
+
+
+class HrrrApi:
+    """Class to interact with to get and download HRRR data. More information
+    about HRRR data can be found at
+    `this link <https://registry.opendata.aws/noaa-hrrr-pds/>`_
+    and
+    `this link <https://www.nco.ncep.noaa.gov/pmb/products/hrrr/>`_
+
+    :param prereise.gather.winddata.hrrr.downloader.Downloader downloader:
+    :param str base_url: url to download data from. Should take "dt", "product"
+        and "hours_forecasted" as format variables. See
+        prereise.gather.winddata.hrrr.constants for more detail
```

Comment:
I think we can use :mod:`prereise.gather.winddata.hrrr.constants` to reference the module here